editor/src/components/inspector/inspector-common.tsx

```diff
@@ -246,6 +251,37 @@ export function widthHeightFromAxis(axis: Axis): 'width' | 'height' {
   }
 }
 
+export function convertWidthToFlexGrow(
+  metadata: ElementInstanceMetadataMap,
+  elementPath: ElementPath,
+  parentPath: ElementPath,
+): Array<CanvasCommand> {
+  const element = MetadataUtils.getJSXElementFromMetadata(metadata, elementPath)
+  if (element == null) {
+    return []
+  }
+
+  const parentFlexDirection =
+    MetadataUtils.findElementByElementPath(metadata, parentPath)?.computedStyle?.[
+      'flexDirection'
+    ] ?? 'row'
```

Comment:
Minor point, but we have an existing function for this that you could alternatively use [`getSimpleFlexDirection`](https://github.com/concrete-utopia/utopia/blob/master/editor/src/core/model/element-metadata-utils.ts#L388) (or `getFlexDirection` immediately above that).